src/mdm-catalogue-item.model.ts

```diff
@@ -105,4 +105,47 @@ export interface AnnotationCreatePayload extends Payload {
 
 export interface AnnotationChildCreatePayload extends Payload {
   description: string;
-}
\ No newline at end of file
+}
+
+/**
+ * Represents all the domain types that can be used with the Mauro path APIs.
+ */
+export type PathableDomainType =
+  'annotations'
+  | 'authorities'
+  | 'classifiers'
+  | 'codeSets'
+  | 'catalogueUsers'
+  | 'dataClasses'
+  | 'dataClassComponents'
+  | 'dataElement'
```

Comment:
dataElements

Reply:
Typo has been fixed. And for your other question, `encodeURIComponent()` will replace with "%20" and not use "+", e.g.

```js
encodeURIComponent('Hello world');

// Output: 'Hello%20world'
```